source/proxy_strategy.c

```diff
@@ -13,12 +13,49 @@
 #    pragma warning(disable : 4221)
 #endif /* _MSC_VER */
 
+/*SA-Added Start*/
 
-char* get_kerberos_usertoken();
-char *get_ntlm_credential();
-char *get_ntlm_response();
-void send_ntlm_challenge_header(size_t length, uint8_t *httpHeader, size_t length1, uint8_t *httpHeader1, size_t num_headers);
-void send_kerberos_https_status(int httpStatusCode);
+#if defined(_MSC_VER)
+#    pragma warning(push)
+#    pragma warning(disable : 4152)
```

Comment:
merge with the above warning disable block

Reply:
This code path is commented now as we are using config to add callback functions.